src/AuthorizationInterface.php

```diff
@@ -13,10 +13,6 @@
 {
     /**
      * Check if a role is granted for the request
-     *
-     * @param string $role
-     * @param ServerRequestInterface $request
-     * @return bool
```

Comment:
Any reference for this? When and where was the decision taken?

Reply:
We discussed it here:
https://discourse.zendframework.com/t/php-7-1-and-phpdocs/171
You can see there @weierophinney and @Ocramius opinions, and @weierophinney already did the same in `zend-expressive-hal`, for example:
https://github.com/zendframework/zend-expressive-hal/blob/master/src/LinkGenerator.php#L43

Also in other place of this library we cen see only function description without tags param/return, for example:
https://github.com/zendframework/zend-expressive-authorization/blob/master/src/ConfigProvider.php#L26

I'm working on it in ZendCodingStandard to make it consistent, but.... it could take another while. If you can read the forum post I've mentioned before, and add there also your opinion it will be great! Thanks!

Reply:
> We discussed it here…

Right, _discussed_ - not more!

The same problem again, like the Travis and Composer file: the forum is good for discussion but not for any reference! Not everyone reads the forum or all comments.

> Also in other place of this library…

The same here: no one can read all repositories!

> I'm working on it in ZendCodingStandard…

The coding standard and your work is great, but this is also no reference!

This is the right place for a reference: https://github.com/zendframework/maintainers/blob/master/CODING_STANDARDS.md
(We should add MkDocs to the maintainers repository.)

---

Don't get me wrong I like the new form of PHPDocs, but without any (official) reference, PR's and reviews will always contain errors!

/cc @weierophinney